src/components/Wallet/Exchange/ExchangeShowMore/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formatProperlyValue } from '../../../../utils/dexFormater';
+import Button from '../../../Button/Button';
+import styles from '../styles.module.scss';
+import { ReservedAssetPropTypes } from '../proptypes';
+
+function ExchangeShowMore({
+  handleModalLiquidity,
+  asset1,
+  asset2,
+  liquidity,
+  asset1ToShow,
+  asset2ToShow,
+  reserved,
+  lpTokensBalance,
+  asset1Decimals,
+  asset2Decimals,
+}) {
+  return (
+    <div className={styles.moreDetails}>
+      <div className={styles.reserved}>
+        {(reserved?.asset2?.isEmpty || reserved?.asset1?.isEmpty) && (
+        <div>
+          Pool don&apos;t have any liquidity
+        </div>
+        )}
+        {reserved && (
+        <div>
+          {`In the pool
+              ${formatProperlyValue(asset1, reserved.asset1, asset1ToShow, asset1Decimals || 0)}  
+            / ${formatProperlyValue(asset2, reserved.asset2, asset2ToShow, asset2Decimals || 0)}` }
+        </div>
+        )}
+      </div>
+      <div className={styles.liquidity}>
+        {liquidity
+          ? `Your liquidity: ${((liquidity / lpTokensBalance) * 100)}% (${liquidity} Lp Tokens)`
```

Comment:
% isn't really important. In a healthy system, all users will have some very small share of the LP.
Number of LP tokens similarly isn't really useful to a user.
What is useful is how many asset1 and asset2 "belongs" to the user.

See https://support.uniswap.org/hc/article_attachments/21158962149645 for an example of how it looks in uniswap v2, on which our dex is loosely based.